app/src/util/types.ts

```diff
@@ -438,3 +443,21 @@ export type UserAccessResponse = {
   isProjectAdmin: boolean;
   isCollaborator: boolean;
 };
+
+export type CityWithProjectDataResponse = {
+  cityId: string;
+  name: string;
+  locode: string;
+  populationYear: number;
+  country: string;
+  population: number;
+  project: {
+    name: string;
+    cityCountLimit: number;
+    organization: {
+      organizationId: string;
+      name: string;
+      contactEmail: string;
+    };
+  };
+};
```

Comment:
Might be good to add a `?` to entries that could be null here so we don't ignore nullabillity issues.
There is not just a single `Population` table entry for each city, there can even be multiple for a single inventory created (because the different population values are not necessarily available from the same year).